spec/feature/user_searches_resources_spec.rb

```diff
@@ -13,10 +14,26 @@
         fill_in 'query', with: title
         find('.navbar__search-button').click
       end
+      wait_for_ajax
       expect(page).to have_text(title)
       expect(page).to have_css('a.btn-add-to-list')
     end
 
+    scenario 'users can see the total Annotations' do
+      title = Faker::Hipster.sentence
+      metadata = { creators: 'Rachel Carson', date: Time.zone.today }
+      create(:resource, title: title, metadata: metadata)
+      wait_for { Resource.search(title).results.total }.to eq(1)
+
+      visit new_search_path
+      within('.customer-search-form') do
+        fill_in 'query', with: title
+        find('.navbar__search-button').click
+      end
+      wait_for_ajax
+      expect(page).to have_text('Annotations')
+    end
+
```

Comment:
Can we also add a spec to see if it actually renders the right # of annotations? We could use VCR to record a sample URL, like so: http://www.metastatic.org/text/This%20is%20Water.pdf

That URL currently has 2 annotations. Then, we can make sure the text says "2 Annotations".

Reply:
@ptrikutam After doing some research I found that is overcomplicated because I need to introduce 2 new dependencies (`VCR` and `puffying_billy`) and make it work together. I think that we don't need that level of assertion for now.

Reply:
Sounds good. Thanks for digging, and agreed -- this is not a critical assertion at this point.